doc/source/structures/misc/time.rst

```diff
@@ -32,6 +32,8 @@ Special variable TIME
 
     Any time you perform arithmetic on **TIME** you get a result back that is also a :struct:`TimeSpan`. In other words, TIME is a :struct:`TimeSpan`, but TIME + 100 is also a :struct:`TimeSpan`.
 
+.. highlight:: none
+
     Note that Kerbals do not have the concept of "months"::
```

Comment:
I would guess that the default highlighting is kerboscript, but we choose to disable it temporarily.